uber/tests/templates/templates_2/test_extends_1.html

```diff
@@ -0,0 +1,6 @@
+{%- import 'test_import_macros.html' as macros -%}
+{%- extends 'test_extends_1.html' -%}
+{%- block main -%}
+{{ super() }}
```

Comment:
Sanity check: if I've understood things properly, then nothing we've done here would stop us from being able to call ``super()`` multiple times for multiple different ``block``s, right?  As far as I can tell, that should work perfectly fine, since it's the ``extends`` which causes the "parent" template to be loaded, and from there the ``super()`` call should just work the way it normally would without any problems.  I just wanted to make sure I had that right.